www-apps/kibana-bin/files/kibana.service

```diff
@@ -3,16 +3,20 @@ Description=Kibana Web Application
 After=network.target
 
 [Service]
-Environment=BABEL_CACHE_PATH=/var/cache/kibana/.babelcache.json
+Environment=BABEL_CACHE_PATH=%C/kibana/.babelcache.json
 
 WorkingDirectory=/opt/kibana
+StateDirectory=kibana
+StateDirectoryMode=0750
+CacheDirectory=kibana
```

Comment:
Don't you need to adjust the OpenRC script as well? Or does it cover creating the directory?

Reply:
OpenRC script has checkpath calls in start_pre() already